src/test/java/uk/gov/hmcts/reform/rsecheck/controller/RootControllerTest.java

```diff
@@ -33,7 +35,8 @@ public void testWelcomeMessage() {
     }
 
     @Test
-    public void testGetPublication() {
+    @DisplayName("Tests that a 200 code is returned, with a non null body")
+    public void testGetPublicationOkResponse() {
```

Comment:
Negative test for if no publication is found?

Reply:
Updated - added a test for this :)